contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -233,11 +233,18 @@ def _take_capture(self, length_in_seconds: int):
         time.sleep(length_in_seconds)
         logging.info('Stopping capture')
         toggle_capture_button.click_input()
+        self._wait_for_capture_completion()
 
     def _verify_existence_of_tracks(self):
         logging.info("Verifying existence of at least one track...")
         MatchTracks(expected_count=1, allow_additional_tracks=True).execute(self.suite)
 
+    def _wait_for_capture_completion(self):
+        logging.info("Waiting for capture to finalize...")
+        wait_for_condition(lambda: self.find_control('Window', 'Finalizing capture',
+                                                     recurse=False, raise_on_failure=False) is None, max_seconds=30)
```

Comment:
Maybe we should increase the max seconds? On my machine it actually took a reasonable amount of time already (didn't measured though).

Reply:
Good point, the video of the failed test shows the dialog for ~16 seconds.

Reply:
You're right. I've changed it to 120 seconds now - better to be safe here, as the tests may run for multiple minutes anyways.